blaze-client/src/main/scala/org/http4s/client/blaze/PooledHttp1Client.scala

```diff
@@ -18,10 +18,14 @@ object PooledHttp1Client {
       maxWaitQueueLimit: Int = bits.DefaultMaxWaitQueueLimit,
       maxConnectionsPerRequestKey: RequestKey => Int = _ => bits.DefaultMaxTotalConnections,
       config: BlazeClientConfig = BlazeClientConfig.defaultConfig): Client[F] =
-    Http1Client.mkClient(
-      config.copy(
-        maxTotalConnections = maxTotalConnections,
-        maxWaitQueueLimit = maxWaitQueueLimit,
-        maxConnectionsPerRequestKey = maxConnectionsPerRequestKey
-      ))
+    Effect[F]
+      .toIO {
+        Http1Client.mkClient(
+          config.copy(
+            maxTotalConnections = maxTotalConnections,
+            maxWaitQueueLimit = maxWaitQueueLimit,
+            maxConnectionsPerRequestKey = maxConnectionsPerRequestKey
+          ))
+      }
+      .unsafeRunSync()
```

Comment:
Honestly I am not ok with this `unsafeRunSync` call if it's litearally the same thing as http1client.

Can we remove this shit altogether? @rossabaker @ChristopherDavenport

Reply:
I mean I'm a fan of enforcing safety by construction. Any reason we should leave this?

Reply:
https://gitter.im/http4s/http4s?at=5b3d61cef1664406611a5117

Hell yeah. Time to set this on fire.

Reply:
🔥 killed!